### PR TITLE
AArch64: Implement vector masked FMA evaluator

### PR DESCRIPTION
**Pull request by Akira1Saitoh**
This commit implements vmfma evaluator.

Signed-off-by: Akira Saitoh <saiaki@jp.ibm.com>

### CODE REVIEW

File: compiler/aarch64/codegen/OMRTreeEvaluator.cpp [hunk 1]
```diff
@@ -2499,7 +2499,80 @@ OMR::ARM64::TreeEvaluator::vmdivEvaluator(TR::Node *node, TR::CodeGenerator *cg)
...
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Node *thirdChild = node->getThirdChild();
+   TR::Node *fourthChild = node->getChild(3);
+
+   TR::Register *firstReg  = cg->evaluate(firstChild);
+   TR::Register *secondReg  = cg->evaluate(secondChild);
+   TR::Register *thirdReg  = cg->evaluate(thirdChild);
```
Comment (knn-k) [inline compiler/aarch64/codegen/OMRTreeEvaluator.cpp:2509]: I don't think you need two whitespaces between `Reg` and `=` in these three lines.

File: compiler/aarch64/codegen/OMRTreeEvaluator.cpp [hunk 2]
```diff
@@ -2499,7 +2499,80 @@ OMR::ARM64::TreeEvaluator::vmdivEvaluator(TR::Node *node, TR::CodeGenerator *cg)
...
+   if (flipMask)
+      {
+      /* BIT inserts each bit from the first source if the corresponding bit of the second source is 1. */
+      generateTrg1Src2Instruction(cg, TR::InstOpCode::vbit16b, node, targetReg, firstReg, maskReg);
+      }
+   else
+      {
+      /* BIF inserts each bit from the first source if the corresponding bit of the second source is 0. */
+      generateTrg1Src2Instruction(cg, TR::InstOpCode::vbif16b, node, targetReg, firstReg, maskReg);
+      }
```
Comment (knn-k) [inline compiler/aarch64/codegen/OMRTreeEvaluator.cpp:2567]: I would suggest introducing a new variable for the bitwise insert operation like `insertOp = flipMask ? TR::InstOpCode::vbit16b : TR::InstOpCode::vbif16b;`

File: compiler/aarch64/codegen/OMRTreeEvaluator.cpp (hunk 1 above)
Comment (Akira1Saitoh) [reply compiler/aarch64/codegen/OMRTreeEvaluator.cpp]: Agreed. Fixed it.

File: compiler/aarch64/codegen/OMRTreeEvaluator.cpp (hunk 2 above)
Comment (Akira1Saitoh) [reply compiler/aarch64/codegen/OMRTreeEvaluator.cpp]: Changed this in the same manner as in https://github.com/eclipse/omr/pull/6738.

**Comment by knn-k** [comment]
Jenkins build aarch64

**Review by knn-k — APPROVED** [review]

**Pull request merged**